at_lookup/example/README.md

```diff
@@ -0,0 +1,11 @@
+<img src="https://atsign.dev/assets/img/@platform_logo_grey.svg?sanitize=true">
+
+### Now for a little internet optimism
+
+# at_lookup_example library
+
+The atlookup package is an interface between the Client SDK and the cloud secondary.
+
```

Comment:
It would be good to have the code snippet that was previously in the README (and main.dart) in this file.

Reply:
Added code snippet.